supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,117 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A doc text is a short description of an engineering Bugzilla or Jira issue that is published in product Errata Advisories and Release Notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with _The IBM Style Guide_. You can write doc texts in more than one way, depending on the issue type: 
+
+* Enhancements
+* Bug Fixes
+* Known Issues
+* Technology Preview
+* Deprecated Functionality
+
+Use the following general structure for consistency.
+
+.Enhancement
+ 
+Use present tense in one of the following formats:
+
+----
+This enhancement <present tense explanation>.
+----
+
+----
+With this update, <present tense explanation>.
+----
+
+Example doc texts:
+----
+“This enhancement optimizes migration of an RBD volume from one Cinder back end to another when the volume resides within the same Ceph cluster. If both volumes are in the same Ceph cluster, Ceph performs data migration instead of the cinder-volume process. This reduces migration time.”
```

Comment:
Can you update the quotes in here to be normal keyboard quotes (for lack of a better term :laughing: )

Reply:
Yes! But now that I look at the build again, these examples are already in code block formatting, so I don't think any quotes are necessary at all.